android/app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
   <uses-permission android:name="android.permission.INTERNET"/>
-  <uses-permission android:name="android.permission.CAMERA"/>
```

Comment:
q: Is this superseeded by something else i.e does this breaks camera perms?

Reply:
Or perhaps this is the rationale?
https://docs.expo.dev/guides/permissions/#android
> Most permissions are added automatically by libraries that you use in your app either with [config plugins](https://docs.expo.dev/config-plugins/plugins-and-mods/#create-a-plugin) or with a package-level AndroidManifest.xml. You only need to use android.permissions to add additional permissions that are not included by default in a library.

Either way, sounds like it wouldn't hurt to keep explicit?

Reply:
Sent back in [19d6b78](https://github.com/shapeshift/mobile-app/pull/109/commits/19d6b78d18a066a2297b173094cca097e6ae30b8)

Let's see if expo remove it with some codemod at some point in the future